strawberryfields/gbs/data.py

```diff
@@ -0,0 +1,217 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+GBS Datasets
+============
+
+**Module name:** :mod:`strawberryfields.gbs.data`
+
+.. currentmodule:: strawberryfields.gbs.data
+
+This module provides access to pre-calculated datasets of simulated GBS samples generated from a
+range of input graphs. Each dataset corresponds to a set of samples generated from a fixed graph
+with corresponding values for:
+
+- ``n_mean``: mean number of photons in the GBS device
+
+- ``n_max``: maximum number of photons allowed in any sample
+
+-  ``threshold``: flag to indicate whether samples are generated with threshold detection or
+   with photon-number-resolving detectors.
+
+- ``n_samples``: total number of samples in the dataset
+
+- ``modes``: number of modes in the GBS device or, equivalently, number of nodes in the graph

```

Comment:
Nitpick, but for the above, I would either have full-stops for all bullet point lists, or no full stops (to keep it consistent).

Reply:
Fixed 😁